### DISCUSSION TITLE
How to improve output quality of animated WebP's?

### DISCUSSION THREAD

**Discussion by mementobuilds** [ImageMagick/ImageMagick#3766] [Help]
Hi, 

I'm trying to overlay an animated webp that's half-transparent "animated_cover.webp" over a png "background.png" to get an animated "output.webp".
I chose webp for its better quality and lower size than a GIF, but the quality seems to be worse than exporting to GIF. Additionally I can see some glitches in the resulting webp, that are not here when exporting to GIF, so I was wondering if there is something missing in my command, or if my magick version lacks some webp configurations? 
One additional problem I noticed is that I cannot use a webp as background (background.webp), but only .png. When using webp, a output.webp with 0 bytes is generated. Any way to solve this? 

Commnad: 
`magick convert background.png -coalesce null: animated_cover.webp -gravity center -background none -layers composite output.webp`

ImageMagick version:
Version: ImageMagick 7.0.11-3 Q16 x64 2021-03-06 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Visual C++: 192829337
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raw rsvg tiff webp xml zip zlib

Thanks!

EDIT: If I add `-define webp:lossless=true` to the command, it outputs a webp file with 0 bytes.

**Comment by dlemstra** [comment]
Can you share  `animated_cover.webp`? And why are you doing `-coalesce` before you are reading the animated webp file?

**Reply by mementobuilds** [reply]
I created a more detailed question here: https://github.com/ImageMagick/ImageMagick/discussions/3771

It turns out that -quality 100 breaks webp and only -quality 99 works. I still get some minor glitches which I cannot explain, but quality issue was solved that way.
